texts/navigation.yml

```diff
@@ -4,7 +4,7 @@ items:
   index: Overview
   data: Data
   content: Content
-  deposit-dates: Deposit dates
+  depositDates: Deposit dates
```

Comment:
Why did this change?

Reply:
that's a leftover. I was just testing something. Thanks